homeassistant/bootstrap.py

```diff
@@ -49,6 +50,13 @@
     webhook as webhook_pre_import,  # noqa: F401
     websocket_api as websocket_api_pre_import,  # noqa: F401
 )
+
+# Ensure group config_flow is imported so it does not need the import
+# executor since config_flows are preloaded when the component is loaded.
+# Even though group is pre-imported above we would have still had to wait
+# for the config flow to be imported when the import executor is the most
+# busy.
+from .components.group import config_flow as group_config_flow  # noqa: F401
```

Comment:
It would have been cleaner to import the config flow in `group`, `__init__.py` but its currently a circular import that needs > 1000 lines of rearranging to unravel.